vbfml/scripts/err_analysis

```diff
@@ -170,15 +196,18 @@ def predict(
 
     predictions = []
     image_pixels = []
+    labels = []
     for ibatch in tqdm(
         range(len(validation_sequences["image"])), desc="Making predictions"
     ):
-        features, _, _ = validation_sequences["image"][ibatch]
-        predictions.append(model.predict(features).argmax(axis=1))
+        features, labels_oneshot, _ = validation_sequences["image"][ibatch]
+        predictions.append(model.predict(features)[:, 1])
```

Comment:
If I understand correctly, you are appending the signal-like scores to the `predictions` array. Can you add a comment here, saying that `[:, 1]` indexing represents that we're taking the signal scores (it is not very intuitive, and hopefully won't lead to mistakes in the future)? Thanks!

Reply:
I added 2 lines of comments